.github/workflows/system-test.yml

```diff
@@ -40,14 +44,10 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
         run: ./scripts/tools/install-deps.sh
-      
+
       - name: Log into Azure
         uses: azure/login@v2
         with:
```

Comment:
You've deleted a bunch of comments that you asked me to add originally, have you changed your mind?

Reply:
accidental delete. Thanks for catching